final_prep/mod_zero_hero.rb

```diff
@@ -56,16 +80,45 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  "name" => "Bob Barker",
+  "smell" => "Bengay and Aspercreme",
+  "weight" => 189,
+  "citiesDestroyed" => ["Tokyo", "Los Angeles", "San Fransisco"],
+  "luckyNumbers" => [76, 32, 1],
+  "address" => {number:"123", street:"Fake Street", state: "CO", zip: "80120"}
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age
+
```

Comment:
Here is where you'll want to declare your instance variables (name, super_power, age...)